mcs_benchmark_data/pipelines/commonsense_qa/roberta_commonsense_qa_submission_transformer.py

```diff
@@ -15,7 +18,15 @@ class RobertaCommonsenseQaSubmissionTransformer(_BenchmarkSubmissionTransformer)
 
     def transform(
         self,
+        *,
+        extracted_path: Path,
+        file_names: RobertaCommonsenseQaSubmissionFileNames,
         **kwds,
     ) -> Generator[_Model, None, None]:
 
-        yield from self._transform(submission_name="roberta", **kwds)
+        yield from self._transform(

```

Comment:
If you pass submission_name up on the constructor instead of here then you can still make _BenchmarkSubmissionTransformer._transform public.